users/config.rst

```diff
@@ -541,9 +594,17 @@ address
 paused
     True if synchronization with this devices is (temporarily) suspended.
 
-allowedNetwork
-    If given, this restricts connections to this device to only this network
-    (see :ref:`allowed-networks`).
+:ref:`allowedNetwork <advanced-device-allowednetworks>`
+    If given, this restricts connections to this device to only this network.
```

Comment:
Link on text instead of option name, please.  Not really discoverable, I wondered what the format should be during review.